<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>4</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>4</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>禁手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>93</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>28</y>
      <width>44</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="Mode_0">
       <property name="text">
        <string>关</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Mode_1">
       <property name="text">
        <string>开</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>28</y>
      <width>70</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Type_1">
       <property name="text">
        <string>人机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Type_2">
       <property name="text">
        <string>机人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Type_3">
       <property name="text">
        <string>人人</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
